llvm/test/CodeGen/AMDGPU/codemodel.ll

```diff
@@ -0,0 +1,9 @@
+; RUN: not llc -verify-machineinstrs -o - -mtriple=amdgcn-unknown-amdhsa -mcpu=gfx900 -code-model=tiny < %s 2>&1 | FileCheck %s --check-prefix=TINY
+; RUN: not llc -verify-machineinstrs -o - -mtriple=amdgcn-unknown-amdhsa -mcpu=gfx900 -code-model=kernel < %s 2>&1 | FileCheck %s --check-prefix=KERNEL
```

Comment:
need test for all -code-model values

Reply:
As matt suggested, this test is not exercising the code changes that I intended in #70760. So I will remove this test.